event_response_tool/event_response.py

```diff
@@ -0,0 +1,323 @@
+if not os.path.exists(config_file):
+    alert.error("Config file not present. Cannot continue.")
+config = configparser.ConfigParser()
+config.read(config_file)
+
+
+class EventResponseHandler:
+    def __init__(self, output_dir, event_response_inputs):
+        self.output_dir = output_dir
+        self.event_response_inputs = event_response_inputs
+
+    def execute(self):
+        self.event_weights_df = read_input_file(
+            self.event_response_inputs.event_weights_csv
+        )
+        if (
+            self.event_response_inputs.layer_views_csv
+            or self.event_response_inputs.portfolio_view_uuid
+        ):
+            modify_layer_loss = LayerLossDuplicator(
+                config=config,
+                output_dir=self.output_dir,
+                event_weights_df=self.event_weights_df,
+                layer_ids_csv=self.event_response_inputs.layer_views_csv,
+                portfolio_uuid=self.event_response_inputs.portfolio_view_uuid,
+            )
+            modify_layer_loss.modify_layer_loss_data()
+        else:
+            ap_creator = AnalysisProfileCreator(
+                config=config,
+                event_weights_df=self.event_weights_df,
+                total_num_of_events=self.event_response_inputs.total_number_of_events,
+                trial_count=self.event_response_inputs.trial_count,
+                catalog_description=self.event_response_inputs.catalog_description,
+                simulation_description=self.event_response_inputs.simulation_description,
+                analysis_profile_description=self.event_response_inputs.analysis_profile_description,
+                old_analysis_profile_uuid=self.event_response_inputs.old_analysis_profile_uuid,
+            )
+            if self.event_response_inputs.old_analysis_profile_uuid:
+                ap_creator.update_analysis_profile()
+            else:
+                ap_creator.create_analysis_profile()
+
+
+# Set up logging
+def init_logging():
+    logging.config.fileConfig("logging.ini")
+    logger = logging.getLogger(__name__)
+
+
+def init_directories():
+    # Create unique output directory for every attempt (FORMAT: archive/event_response-[datetime])
+    # to store the output files and log
+    now = datetime.now()
+    current_datetime_string = now.strftime("%d-%m-%Y-%H:%M:%S")
+    target_dir = "{}-{}".format("event_response", current_datetime_string)
+    output_directory = config.get("defaults", "output_directory")
+    full_output_dir = "{}/{}".format(output_directory, target_dir)
+
+    try:
+        os.makedirs(full_output_dir, exist_ok=True)
+    except OSError as e:
+        alert.exception(
+            f"Creation of {full_output_dir} directory failed: {e}"
+        )
+    else:
+        if os.path.isfile("activity.log"):
+            shutil.move("activity.log", f"{full_output_dir}/activity.log")
+        return full_output_dir
+
+
+def check_empty_credentials(url, user_name, password):
```

Comment:
```suggestion
def collect_credentials(url, user_name, password):
```
or something is maybe more clear? We aren't validating the credentials here, and if both the parameter and the config are empty, we aren't doing anything about it here.

Reply:
Is this function being used? `check_and_set_optional_inputs` seems to do this work for these parameters and the others